trade-finance-manager-ui/server/test-mocks/mock-banks.ts

```diff
@@ -74,7 +74,7 @@ const MOCK_BANKS = {
       { startMonth: 9, endMonth: 11 },
       { startMonth: 12, endMonth: 2 },
     ],
-  },
-};
+  } as const,
+} as const;
```

Comment:
What are we trying to achieve by adding `as const` everywhere here?

Reply:
I originally did it because I was exporting the object incorrectly, but I think adding it once at the bottom will achieve the same effect now :)

Reply:
Would still be interested to understand why we're adding this just at the end here. Is it just to make the properties read only? I don't think we're not deriving any types from it so I don't think it's that?